packages/vscode-extension/package.json

```diff
@@ -0,0 +1,263 @@
+  },
+  "categories": [
+    "Other"
+  ],
+  "activationEvents": [
+    "onStartupFinished"
+  ],
+  "main": "./build/extension/extension.js",
+  "contributes": {
+    "commands": [
+      {
+        "command": "traces.openTrace",
+        "title": "Open Trace",
+        "icon": "assets/media/dep.svg"
+      },
+      {
+        "command": "outputs.openOutput",
+        "title": "Open Output",
+        "icon": "assets/media/chart-line-solid.svg"
+      },
+      {
+        "command": "traces.openTraceFile",
+        "title": "Open Trace",
+        "icon": "assets/media/dep.svg"
+      }
+    ],
+    "viewsContainers": {
+      "activitybar": [
+        {
+          "id": "trace-explorer",
+          "title": "Trace Explorer",
+          "icon": "assets/media/chart-line-solid.svg"
+        }
+      ]
+    },
+    "views": {
+      "trace-explorer": [
+        {
+          "id": "traces",
+          "name": "Traces",
+          "icon": "assets/media/dep.svg"
+        },
+        {
+          "id": "analysis",
+          "name": "Analysis",
+          "icon": "assets/media/chart-line-solid.svg"
+        }
+      ]
+    },
+    "viewsWelcome": [
+      {
+        "view": "traces",
+        "contents": "No traces found [learn more](https://www.eclipse.org/tracecompass/).\n[Add traces](command:nodeDependencies.addEntry)"
+      },
+      {
+        "view": "analysis",
+        "contents": "No trace selected [learn more](https://www.eclipse.org/tracecompass/)"
+      }
+    ],
+    "menus": {
+      "view/item/context": [
+        {
+          "command": "traces.openTrace",
+          "when": "view == traces",
+          "group": "inline"
+        },
+        {
+          "command": "outputs.openOutput",
+          "when": "view == analysis",
+          "group": "inline"
+        }
+      ],
+      "explorer/context": [
+        {
+          "command": "traces.openTraceFile",
+          "group": "navigation"
+        }
+      ]
+    },
+    "configuration": {
+      "title": "Trace Compass",
+      "properties": {
+        "trace-compass.traceserver.url": {
+          "type": "string",
+          "default": "http://localhost:8080",
+          "description": "Enter the trace server's URL, including port. Eg: http://localhost:8080."
+        },
+        "trace-compass.traceserver.apiPath": {
+          "type": "string",
+          "default": "tsp/api",
+          "description": "Enter the trace server's API path, to be appended to the server URL. Eg: 'tsp/api'."
+        }
+      }
+    }
+  },
+  "devDependencies": {
+    "@babel/core": "7.9.0",
+    "@svgr/webpack": "4.3.3",
+    "@types/glob": "^7.1.1",
+    "@types/mocha": "^7.0.2",
+    "@types/node": "^13.11.0",
+    "@types/react": "^16.9.43",
+    "@types/react-dom": "^16.9.8",
+    "@types/vscode": "^1.47.0",
+    "@typescript-eslint/eslint-plugin": "^2.30.0",
+    "@typescript-eslint/parser": "^2.30.0",
+    "babel-eslint": "10.1.0",
+    "babel-jest": "^24.9.0",
+    "babel-loader": "8.1.0",
```

Comment:
I see this dependency not being found when building the `browser example app`. Maybe it should be a runtime dependency. but most likely it's not a good idea to mix Theia and vscode stuff under the same monorepo, with shared hoisted dependencies. 

Since the Theia example app and the vscode trave viewer extension have no build-time relation, they should be be built separately. They can still be in the same repo though, just not in the same yarn workspace.